### ISSUE TITLE
Weird failure in test_speed on NonStop

### ISSUE THREAD

**Issue by rsbeckerca** [openssl/openssl#25203]
This just started...
```
speed: async_jobs specified but async not supported
speed: Use -help for summary.
../../util/wrap.pl ../../apps/openssl speed -testmode -async_jobs 1 => 1
not ok 14 - Test the async_jobs option
```
I'm not sure what to do with this situation.

**Comment by t8m** [comment]
We just need to skip the testcase in the test_speed recipe.

**Comment by t8m** [comment]
Fix in #25203
